Daily Acceptance Test Failed: TestAccSakuraCloudServer_withoutShutdown

以下のエラーで失敗する。

```console
  --- FAIL: TestAccSakuraCloudServer_withoutShutdown (299.32s)
      testing.go:654: Step 1 error: Check failed: Check 2/2 error: unexpected shutdown has happened
```

しかし手元のMac/Linuxからだと再現しない。

6/25 再発

```
  --- FAIL: TestAccSakuraCloudServer_withoutShutdown (238.18s)
01:52:18
      testing.go:654: Step 1 error: Check failed: Check 2/2 error: unexpected shutdown has happened
```

優先度をあげて対応する。

--- a/sakuracloud/resource_sakuracloud_server_test.go
+++ b/sakuracloud/resource_sakuracloud_server_test.go
@@ -118,8 +118,12 @@ func TestAccSakuraCloudServer_withoutShutdown(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testCheckSakuraCloudServerExists(resourceName, &updated),
 					func(state *terraform.State) error {
-						if created.InstanceStatusChangedAt != updated.InstanceStatusChangedAt {
-							return errors.New("unexpected shutdown has happened")
+						if !created.InstanceStatusChangedAt.Equal(updated.InstanceStatusChangedAt) {
+							return fmt.Errorf(
+								"unexpected shutdown has happened: ChangeAt: before: %s after: %s",
+								created.InstanceStatusChangedAt.String(),
+								updated.InstanceStatusChangedAt.String(),
+							)
 						}
 						return nil
 					},
@@ -130,7 +134,7 @@ func TestAccSakuraCloudServer_withoutShutdown(t *testing.T) {
 				Check: resource.ComposeTestCheckFunc(
 					testCheckSakuraCloudServerExists(resourceName, &editParamChanged),
 					func(state *terraform.State) error {
-						if updated.InstanceStatusChangedAt == editParamChanged.InstanceStatusChangedAt {
+						if updated.InstanceStatusChangedAt.Equal(editParamChanged.InstanceStatusChangedAt) {
 							return errors.New("expected shutdown has not happened")
 						}
 						return nil

